robottelo/ui/base.py

```diff
@@ -474,6 +474,7 @@ def field_update(self, loc_string, newtext):
         """
         Function to replace the existing/default text from textbox
         """
+        self.wait_until_element(locators[loc_string])
```

Comment:
Don't use that method at all and use `assign_value` instead. Or as last resort `text_field_update`